### PR TITLE
Convert wiki to UNIX format and refresh most content

### PR DESCRIPTION
**Pull request by theopolis**
1. Refresh the documentation using a small style guide.
2. Add a unit test for the example config.

### CODE REVIEW

**Comment by theopolis** [comment]
ok to test

**Comment by theopolis** [comment]
I'm working on integrating some of the helpful notes from [Windmill's README](https://github.com/heroku/windmill/blob/master/README.md).

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+- **Installation**: Deep dives into OS-specifics, packaging, and switches that control starting tools.
+- **Deployment**: Tool concepts and all the wonderful goodies of making osquery useful.
+- **Development**: Help and guides for starting with osquery development and build.
+
+Make a new "filename.md" within the category folder withing `/docs/wiki/CATEGORY/`. Then add the friendly page title and path to [mkdocs.yml](https://github.com/facebook/osquery/blob/master/mkdocs.yml), in the order the page should appear within the wiki sidebar.
+
+### Wiki style tips
+
+- Inline code highlighing (`$ echo 'this is inline'`) does not look the best in RTD, try to have as little inline syntax highlighting as possible.
+- **osqueryd**, **osqueryi** and other tool names should be in bold. Use `inline highligh` when a tool or script is mentioned for the first time.
```
Comment (sharvilshah) [inline docs/README.md]: nit: typo in `inline highligh`

File: docs/wiki/deployment/log-aggregation.md [hunk 2]
```diff
@@ -74,19 +68,17 @@ sourcetype = osquery_warning
 
 ### Rsyslog
 
-[rsyslog](http://www.rsyslog.com/) is a tried and testing unix log forwarding service. If you're deploying osqueryd in a production Linux environment where you don't have to worry about lossy network connections, this may be your best option.
+[rsyslog](http://www.rsyslog.com/) is a tried and testing UNIX log forwarding service. If you are deploying **osqueryd** in a production Linux environment where you do not have to worry about lossy network connections, this may be your best option.
 
 ## Analyzing logs
 
-The way in which you analyze logs is very dependent on how you're aggregating logs. At the end of the day, osquery produces results logs in JSON format, so the logs are very easy to analyze on most modern backend log aggregation platforms.
+The way in which you analyze logs is very dependent on you aggregate logs. At the end of the day, osquery produces results logs in JSON format, so the logs are very easy to analyze on most modern backend log aggregation platforms.
```
Comment (sharvilshah) [inline docs/wiki/deployment/log-aggregation.md]: nit: Should be `on how you aggregate logs` or something like: `... very depended on how logs are aggregated`.

File: docs/wiki/index.md [hunk 3]
```diff
@@ -1,23 +1,21 @@
...
+If you're interested in **deploying osquery** to provide your organization with deeper insight into your Linux and OS X hosts check out the [using osqueryd guide](introduction/using-osqueryd.md).
+If you're interested in **performing ad-hoc queries**, check out [using osqueryi](introduction/using-osqueryi.md).
+
+If you're interested in **extending one of the existing osquery tools** or improving core libraries, read the developer documentation pages. You should start with "[building the code](development/building.md)" and "[contributing code](development/contributing-code.md)".
+
+If you're interested in **integrating osquery** into your own tool, check out the [osquery SDK](development/osquery-sdk.md).
+
+## Getting help
+
+If you any part of osquery is not working as expected, please create a [GitHub Issue](https://github.com/facebook/osquery/issues). Keep in touch with osquery developers and users in **#osquery** on **freenode**.
```
Comment (sharvilshah) [inline docs/wiki/index.md]: nit: Should be `If any part of osquery ...`

File: docs/wiki/introduction/overview.md [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
+
+The interactive query console, **osqueryi**, gives you a SQL interface to try out new queries and explore your operating system. With the power of a complete SQL language and dozens of useful tables built-in, **osqueryi** is an invaluable tool when performing incident response, diagnosing a systems operations problem, troubleshooting a performance issue, etc.
+
+osquery is cross platform. Even though osquery takes advantage of very low-level operating system APIs, you can build and use osquery on Mac OS X, Ubuntu, Cent OS and other popular enterprise Linux distributions. This has the distinct advantage of allowing you to be able to use one platform for monitoring complex operating system state across your entire infrastructure. Monitor your corporate Mac OS X clients the same way you monitor your production Linux servers.
```
Comment (sharvilshah) [inline docs/wiki/introduction/overview.md]: nit: CentOS should be one word

**Comment by theopolis** [comment]
Thanks a ton for the review @sharvilshah!

**Pull request merged**